types/forge-apis/index.d.ts

```diff
@@ -190,6 +190,33 @@ export interface JobPayload {
     misc?: JobPayloadMisc | undefined;
 }
 
+export interface JobSvf2OutputPayload {
```

Comment:
`JobSvf2OutputPayload` and `JobSvfOutputPayload` seem to be the same interface as well. Is there a reason why you haven't used a type alias, e.g. `type JobSvf2OutputPayload = JobSvfOutputPayload`? I think keeping those as two different interfaces only makes sense if we expect one of them to change soon.